rules/windows/process_creation/win_apt_bear_activity_gtr19.yml

```diff
@@ -17,11 +17,11 @@ logsource:
     product: windows
 detection:
     selection1:
-        Image: '*\xcopy.exe'
-        CommandLine: '* /S /E /C /Q /H \\*'
+        Image|endswith: '\xcopy.exe'
+        CommandLine|contains: '/S /E /C /Q /H \\'
```

Comment:
could you please also change the spaces stuff here and on line 24 of this rule?

Reply:
Done